releasenotes/notes/cqm-__str__-method-b9f37742662c0810.yaml

```
@@ -0,0 +1,3 @@
+---
+features:
+  - Add ``ConstrainedQuadraticModel.__str__()`` method.
```

It's nice to add issue references when we have them.

```suggestion
  - Add ``ConstrainedQuadraticModel.__str__()`` method.  See  #1188 <https://github.com/dwavesystems/dimod/issues/1188>`_.
```

---

✔️ Did that